packages/core/upload/tests/admin/folder-structure.test.e2e.js

```diff
@@ -26,6 +26,20 @@ describe('Folder structure', () => {
     strapi = await createStrapiInstance();
     rq = await createAuthRequest({ strapi });
 
+    // delete all possibly existing folders
+    const res = await rq({
+      method: 'GET',
+      url: '/upload/folders?pagination[pageSize]=100',
```

Comment:
Discussed it briefly on Slack: should we use 
```suggestion
      url: '/upload/folders?pagination[pageSize]=-1',
```

instead to make it clear there is no hard limit on 100 folders?